spec/features/user_adds_review_to_micro_hood_spec.rb

```diff
@@ -51,4 +51,22 @@
     click_button('Add Review')
     expect(page).to have_content 'User must be signed in'
   end
+
+  scenario 'authorized user tries to submit a rating over 5' do
+    microhood = FactoryGirl.create(:microhood)
+    user = FactoryGirl.create(:user)
+    login_as(user, scope: :user)
+    visit "/microhoods/#{microhood.id}"
```

Comment:
To make your test less-brittle, you should be using the path helpers here (e.g. `visit microhood_path(microhood)`, or even just `visit microhood`) rather than specifying the relative path.